src/uproot/models/RNTuple.py

```diff
@@ -289,25 +293,42 @@ def pagelist(self, listdesc):
         pages = listdesc.reader.read(listdesc.chunk, local_cursor, listdesc.context)
         return pages
 
-    def read_pagedesc(self, destination, start, stop, desc, dtype):
-        num_elements = desc.num_elements
+    def read_pagedesc(self, destination, desc, num_elements, dtype):
         loc = desc.locator
         cursor = uproot.source.cursor.Cursor(loc.offset)
         context = {}
-        uncomp_size = num_elements * dtype.itemsize
+        # bool in RNTuple is always stored as bits
+        isbit = dtype == bool
```

Comment:
Here: is it ever possible for `dtype` to be `np.bool_`?

```python
>>> np.bool_ == bool
False
>>> np.dtype(np.bool_) == bool
True
>>> np.dtype(np.bool_) == np.bool_
True
```

Reply:
right, I was using `np.bool` and numpy gave me a deprecation warning, I think because I got the type from the `_rntuple_num_to_dtype_dict` I have full control over the type

Reply:
NumPy used to include the Python `bool` in its namespace, but that only increased the confusion. The deprecation warning is just about getting it from that namespace, not about using it in arrays, though I'd avoid the use of Python types when defining array types in anything formal.